Portal/Services/WasteService.cs

```diff
@@ -136,5 +137,18 @@ public async Task SaveSelectedWasteType(int journeyId, string selectedWasteType)
 
             await _httpWasteService.SaveSelectedWasteType(journeyId, selectedWasteType);
         }
+
+        public async Task<WasteRecordStatusViewModel> GetWasteRecordStatus(int journeyId)
+        {
+            var result = await _httpWasteService.GetWasteRecordStatus(journeyId);
+            return new WasteRecordStatusViewModel
```

Comment:
Probably worth using auto mapper here isn't it?

Reply:
Yeah, I wanted to get this out the door in the current sprint, so will add automapper in the next story.